.pre-commit-config.yaml

```diff
@@ -13,7 +13,7 @@ repos:
   - id: check-added-large-files
 
 - repo: https://github.com/psf/black
-  rev: "22.3.0"
+  rev: "21.12b0"
```

Comment:
I think this line was auto-added by pre-commit to sync with poetry, that's my fault. Cal you run `poetry update` and commit the changes in lockfile & `pre-commit-config.yaml`` ? That will fix the CI.

Reply:
I did that, but `poetry update` changed the file a lot. I am not quite sure if this is how it should be. I have committed the changes, but feel free to revert the changes or let me know if you want me to delete the commit.

Reply:
Nah, updating the lockfile should never be an issue. If it was, it would mean that our declared versions in pyproject.toml are incorrect.